.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+ - "node"
+ - "8"
+before_script:
+   - npm install -g babel-jest && npm install jest-cli
```

Comment:
I don't see the need for this before_script step because:
1. `babel-jest` and `jest` are already part of devDependencies
1. NODE_ENV is **not** set to production so dev dependencies  will get install when travis runs `npm install`

Reply:
thank @benbraou . I will make a new pr